src/components/search-bar/search-bar.css

```diff
@@ -0,0 +1,11 @@
+.search-bar {
+  border-bottom: 1px solid #ccc;
+  margin-bottom: 0.5em;
+}
+  
+.search-bar-input {
+  border: none;
+  font-size: 1.25em;
+  padding: 0.5em 15px;
+  width: 100%;
+}
```

Comment:
Also super minor: making sure files end with a line break can help a lot of linux/unix tools work a little better. If you install an [EditorConfig](https://editorconfig.org/) plugin for your editor, we have it configured to do this automatically for you. :)

Reply:
Done!